src/sentry/issues/auto_source_code_config/code_mapping.py

```diff
@@ -541,3 +571,35 @@ def find_roots(frame_filename: FrameInfo, source_path: str) -> tuple[str, str]:
     # validate_source_url should have ensured the file names match
     # so if we get here something went wrong and there is a bug
     raise UnexpectedPathException("Could not find common root from paths")
+
+
+def get_path_from_module(module: str, abs_path: str) -> tuple[str, str]:
+    """This converts a Java module name and filename into a real path."""
+    if abs_path.find("$") > -1:
+        raise DoesNotFollowJavaPackageNamingConvention
+
+    parts = abs_path.split(".")
+    if len(parts) != 2:  # file name + extension
```

Comment:
Instead of aborting if `abs_path` contains no `.` can we simply use `module` (with processing below) instead?